index.php

```diff
@@ -0,0 +1,273 @@
+            // Decode the JSON response data into an associative array.
+            $data = json_decode($data, true);
+            //Check if email deliverability is not "DELIVERABLE" and it's not a disposable email or has an invalid format.
+            if ($data['deliverability'] != "DELIVERABLE" && $data['is_disposable_email']['value'] != "false" && $data['is_valid_format']['text'] != "true") {
+              // Set an error message for invalid email
+              $emailErr = "* Invalid email.";
+              // Return an empty string to indicate validation failure.
+              return "";
+            }
+            // If email is valid, set success messages and return the email address.
+            $emailMsg = "Your email id is: $email";
+            $emailSuccess = "email validated successfully.";
+            return $email;
+          }
+        }
+
+    }
+
+    // Calling ValidatName function to validate name and store in variable.
+    $fname = validate($_POST["fname"], $regex, $fnameErr, "First Name");
+    $lname = validate($_POST["lname"], $regex, $lnameErr, "Last Name");
+    $num = validate($_POST["number"], $numRegex, $numErr, "Phone Number");
+    $email = validateEmail($_POST["email"], $emailErr, $emailSuccess, $emailMsg);
+
+    $numMsg = "";
+    //Checking Phone number is 10 digits or not.
+    if (!empty ($num) && strlen($num) == 10) {
+      $num = "+91 {$num}";
+      $numMsg = 'Your Phone Number is: ' . $num;
+    }
+    else {
+      $numErr = "Invalid Phone Number";
+    }
+
+    //Storing Image file.
+    if (!empty ($_FILES['image']['name'])) {
+      $img_temp_name = $_FILES['image']['tmp_name'];
+      $img_name = $_FILES['image']['name'];
+      move_uploaded_file($img_temp_name, "uploads/$img_name");
+    }
+
+    // Declearing and initializing var.
+    $tableErr = false;
+    $tableErrMsg = "";
+
+    //Checking if first name or last name is not empty.
+    if (!empty ($fname) && !empty ($lname)) {
+      //Creating an object .
+      $user = new User($fname, $lname);
+      $fullName = $user->getFname() . ' ' . $user->getLname();
+      $message = "Hello $fullName.";
+
+      //checking if marks field is not emptpy.
+      if (!empty ($_POST['sub'])) {
+        $marksArray = explode("\n", $_POST["sub"]);
+        $tableHead = "Marks";
+        $table = "<table class = 'table' border = '1' >";
+        $table .= "<tr><th>Subject</th><th>Marks</th></tr>";
+
+        // Loop through each subject and marks pair.
+        foreach ($marksArray as $mark) {
+          // Splitting subject and marks pair.
+          $marks = explode("|", $mark);
+          if (count($marks) == 2) {
+            // Adding table rows to the HTML string.
+            $table .= "<tr>";
+            if (is_numeric($marks[0])) {
+              $table .= "<td>$marks[1]</td>";
+              $table .= "<td>$marks[0]</td>";
+            }
+            else if (is_numeric($marks[1])) {
+              $table .= "<td>$marks[0]</td>";
```

Comment:
Couldn't we handle these table data from HTML directly instead of creating a string table?